LICENSE

```diff
@@ -0,0 +1,287 @@
+                      EUROPEAN UNION PUBLIC LICENCE v. 1.2
```

Comment:
just going to presume this license is copy pasted from LDES server or LDIO ? So that i don't have to read it :p

Reply:
That's right :P